src/js/components/Select/Select.js

```diff
@@ -233,7 +233,7 @@ const Select = forwardRef(
     // element to show, trumps inputValue
     const selectValue = useMemo(() => {
       if (valueLabel instanceof Function) {
-        if (value) return valueLabel(value);
+        if (value !== '' && value !== null) return valueLabel(value);
```

Comment:
There are other 'falsey' values like `undefined` or `false` that this handled before. Should we be calling valueLabel for `undefined` or `false`? Might be better to change this to:
```javascript
if (value || value === 0) return valueLabel(value)
```

Reply:
Not sure if this is a use case, but is it possible that someone might have some options like `[true, false]` ?

Reply:
^ good point, we do support having `[true, false]` as options. Maybe we could do something like `if (value || value === 0 || value === false) return valueLabel(value);`? Not sure if there is a cleaner way to write this to get our desired behavior.

Reply:
Cool, just pushed up that change